templates/preview-model.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html lang=en>
+  <head>
+    <meta charset=utf-8>
+    <script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.1.1/model-viewer.min.js"></script>
+  </head>
+  <body>
+    <model-viewer src="/content/{{ self.inscription_id }}" shadow-intensity="1" camera-controls="true" auto-rotate="true" controls="true" action="pan-y" ar-status="not-presenting" style="width: 100%; min-height: 400px; overflow: hidden;"></model-viewer>
```

Comment:
```suggestion
    <model-viewer src=/content/{{ self.inscription_id }} camera-controls=true auto-rotate=true></model-viewer>
```

And let's move the style into a `<style>` tag:

```
<style>
  model-viewer {
    width: 100%;
  }
</style>
```